### PR TITLE
nrf-connect 5.0.2 (new cask)

### PR DESCRIPTION
**Pull request by nbadal**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/search?q=repo%3AHomebrew%2Fhomebrew-cask+is%3Aclosed+is%3Aunmerged+&type=pullrequests) (add your cask's name to the end of the search field).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/n/nrf-connect.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  arch arm: "arm64", intel: "x64"
+
+  version "5.0.2"
+  sha256 arm:   "b691cc5692a818cb067fe79cea6fd4573b9602013d90d9bc9fe2e95649c9eace",
+         intel: "499a9efbec65a62be9e90d23f2505fbafb46702171f1b0180f2d01f81fe7a857"
+
+  url "https://nsscprodmedia.blob.core.windows.net/prod/software-and-other-downloads/desktop-software/nrf-connect-for-desktop/#{version.dots_to_hyphens}/nrfconnect-#{version}-#{arch}.dmg",
+      verified: "nsscprodmedia.blob.core.windows.net/"
+  name "nRF Connect for Desktop"
+  desc "Cross-platform tool framework for assisting development on BLE devices"
```
Comment (bevanjkay) [inline Casks/n/nrf-connect.rb:11]:
```suggestion 
 desc "Framework for development on BLE devices"

File: Casks/n/nrf-connect.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  homepage "https://www.nordicsemi.com/Products/Development-tools/nRF-Connect-for-Desktop"
+
+  livecheck do
+    url "https://www.nordicsemi.com/Products/Development-tools/nRF-Connect-for-Desktop/Download"
+    regex(%r{/nrfconnect-([0-9.]*)-}i)
```
Comment (bevanjkay) [inline Casks/n/nrf-connect.rb:16]:
```suggestion
  regex(/nrfconnect[._-]v?(\d+(?:\.\d+)+)[._-]#{arch}\.dmg/i)

**Review by bevanjkay — COMMENTED** [review]
Please try to add a [zap stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap). The linked documentation provides some recommended methods to create it.

File: Casks/n/nrf-connect.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+  url "https://nsscprodmedia.blob.core.windows.net/prod/software-and-other-downloads/desktop-software/nrf-connect-for-desktop/#{version.dots_to_hyphens}/nrfconnect-#{version}-#{arch}.dmg",
+      verified: "nsscprodmedia.blob.core.windows.net/"
+  name "nRF Connect for Desktop"
+  desc "Framework for development on BLE devices"
+  homepage "https://www.nordicsemi.com/Products/Development-tools/nRF-Connect-for-Desktop"
+
+  livecheck do
+    url "https://www.nordicsemi.com/Products/Development-tools/nRF-Connect-for-Desktop/Download"
+    regex(/nrfconnect[._-]v?(\d+(?:\.\d+)+)[._-]#{arch}\.dmg/i)
```
Comment (khipp) [inline Casks/n/nrf-connect.rb:16]:
```suggestion
    url :url
    strategy :github_latest
```

File: Casks/n/nrf-connect.rb [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+  end
+
```
Comment (khipp) [inline Casks/n/nrf-connect.rb:18]:
```suggestion

  auto_updates true
```

**Review by khipp — COMMENTED** [review]
Upstream publishes releases in a GitHub repository, which is also used by the app's updater. Mirroring the source should make the cask's `url` and `livecheck` less likely to break in the future.

**Review by bevanjkay — APPROVED** [review]

**Pull request merged**